lib/realtime_signs_web/controllers/monitoring_controller.ex

```diff
@@ -11,4 +11,10 @@ defmodule RealtimeSignsWeb.MonitoringController do
     monitor_device_uptime(nodes, timestamp)
     send_resp(conn, 200, "")
   end
+
+  def run_message_log_job(conn, %{"date" => date} = _params) do
```

Comment:
Maybe a `@doc` comment here at least explaining the format of the `date` expected?